### ISSUE TITLE
Rendering issue in IE11

### ISSUE THREAD

**Issue by morningdew830** [femioladeji/react-slideshow#33]
This is what I am seeing in IE11.

![image](https://user-images.githubusercontent.com/22661536/57249559-dd6f7880-7077-11e9-8a70-c67129c0c92a.png)

Could someone fix this issue?

**Comment by morningdew830** [comment]
For referrer,
```
... ... ...
{
    key: 'applySlideStyle',
    value: function applySlideStyle() {
      var _this3 = this;

      this.allImages.forEach(function (eachImage, index) { // <-- this.allImages is an object, not array in ie
        eachImage.style.width = _this3.width + 'px';
      });
    }
  }, {
    key: 'moveSlides',
    value: function moveSlides(_ref) {
... ... ...
```

**Comment by femioladeji** [comment]
Ok, thanks for pointing this out @morningdew830. I'll look into it and get back to you

**Comment by femioladeji** [comment]
I've raised a PR to fix this #34. You'll have to uninstall the package and re-install it. Let me know if the fix worked. Thanks
